python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -1468,12 +1468,14 @@ def setDefaults(self):
         # Explicitly restating because ratio with detect.nSigmaToGrow matters
         self.detectTemplate.nSigmaToGrow = 2.4
         # Higher thresholds make smaller and fewer protected zones around
-        # bright stars. Sources with snr < 50 tend to subtract OK empirically
-        self.detectTemplate.thresholdValue = 50
-        self.detectTemplate.doTempLocalBackground = False
-        self.detectTemplate.reEstimateBackground = False
+        # bright stars. Sources with snr < 300 tend to subtract OK empirically
+        self.detectTemplate.thresholdValue = 300
+        self.detectTemplate.doTempLocalBackground = True
+        self.detectTemplate.reEstimateBackground = True
```

Comment:
It is not for this ticket, but we could replace the background subtraction built in to the detection task with the version I wrote for `subtractImages` that allows for outlier rejection and does a better job around artifacts and bright stars.